### PR TITLE
Feature/validation fix

### PR DESCRIPTION
**Pull request by utas-tajona**

### CODE REVIEW

File: ui/src/components/import/ColumnDef.js [hunk 1]
```diff
@@ -284,49 +343,73 @@ export const ExtendedSize = [
...
     },
     {
         field: '550',
         editable: true,
+        width: 105,
+
     },
     {
-        field: '600',
+        field: '800',
```
Comment (jonescc) [inline ui/src/components/import/ColumnDef.js:362]: Was this intentional?   There's an 800 below

Comment (utas-tajona) [reply ui/src/components/import/ColumnDef.js]: fixed , mistake

**Review by jonescc — APPROVED** [review]
LGTM

**Comment by jonescc** [comment]
@nspool - any comments?

**Comment by nspool** [comment]
@jonescc running it locally and edits are working. Looks good.

**Pull request merged**